test/SourceKit/CodeExpand/code-expand-rdar77665805.swift

```diff
@@ -0,0 +1,32 @@
+// BEGIN main.swift
+enum E { case foo, bar }
+func foo(x: (E) -> Void) {}
+func test() {
+  foo(x: <#T##(E) -> Void#>)
+}
+
+// BEGIN expand.json.in
```

Comment:
I suggest we add a new sourcekitd-test request for expanding a single placeholder by location.